app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -162,7 +165,7 @@ fun MapScreen(
                     // Set the text and the button of the view annotation
                     ItemCalloutViewBinding.bind(viewAnnotation).apply {
                       textNativeView.text =
-                          "Capacity is ${parkingDeserialized.capacity.description}"
+                          screenCapcityString.format(parkingDeserialized.capacity.description)
```

Comment:
Missing a in `screenCapacityString` here

Reply:
Good catch ! Will be add in the next commit.